src/provider/tag-pane.ts

```diff
@@ -37,7 +50,16 @@ export class TagPaneProvider extends ProviderBase {
    * Get the root tag pane items from the tag pane view. This property is not documented.
    */
   private getTagItems(tagPane: WorkspaceLeaf): TagExplorerItem[] {
-    return Object.values((tagPane.view as any).tagDoms) as TagExplorerItem[];
+    return Object.values((tagPane.view as TagPaneView).tagDoms).map((tagDom: TagDom) => this.convertDomToTagItem(tagDom));
+  }
+
+  private convertDomToTagItem(tagDom: TagDom): TagExplorerItem {
+    return {
+        tag: tagDom.tag,
+        collapsed: tagDom.collapsed,
+        setCollapsed: tagDom.setCollapsed.bind(tagDom),
+        children: tagDom.vChildren._children.map((child: TagDom) => this.convertDomToTagItem(child))
```

Comment:
So this is really the core of the change, right? using `vChildren._children` instead of just `children`. What's the difference between the two?

Reply:
I don't know if `children` was intentional. If not, then we could just modify existing type instead of adding a new one